fenix/app/src/main/res/values/styles.xml

```diff
@@ -320,8 +320,8 @@
         <item name="homeBackground">@drawable/private_home_background_gradient</item>
         <item name="bottomBarBackground">@drawable/private_home_bottom_bar_background_gradient</item>
         <item name="bottomBarBackgroundTop">@drawable/private_home_bottom_bar_background_gradient_top</item>
-        <item name="privateBrowsingButtonBackground">@color/fx_mobile_private_text_color_primary</item>
-        <item name="privateBrowsingButtonAccent">@color/fx_mobile_private_layer_color_1</item>
+        <item name="mozac_ic_private_mode_circle_fill_background_color">@color/photonWhite</item>
+        <item name="mozac_ic_private_mode_circle_fill_icon_color">@color/fx_mobile_private_layer_color_2</item>
```

Comment:
Color reference: https://www.figma.com/file/Ha2T4I0nX0h7B95GYxH6wV/Simplified-UI-for-Private-Mode?node-id=4779%3A6865&mode=dev